### PR TITLE
Fix duped alerts

### PR DESCRIPTION
**Pull request by leigh-mil**
## Description
Removes duplicate flash messages for  logout and errors on TO form. 

## Pivotal
https://www.pivotaltracker.com/story/show/166683697

### CODE REVIEW

File: atst/domain/auth.py [hunk 1]
```diff
@@ -61,7 +60,6 @@ def logout():
     if session.get("user_id"):  # pragma: no branch
         del session["user_id"]
         del session["last_login"]
-        formatted_flash("logged_out")
```
Comment (drummond-dds) [inline atst/domain/auth.py]: I think I had issues with the logout message not displaying when a session timed out. This was added to fix [this Pivotal Tracker ticket](https://www.pivotaltracker.com/story/show/166472937). Can you double check that this doesn't break the previous bug?

Comment (drummond-dds) [inline atst/domain/auth.py]: I think I had issues with the logout message not displaying when a session timed out. This was added to fix [this Pivotal Tracker ticket](https://www.pivotaltracker.com/story/show/166472937). Can you double check that this doesn't break the previous bug?

Comment (drummond-dds) [reply atst/domain/auth.py]: You can change the `PERMANENT_SESSION_LIFETIME` to something like `5` seconds to test this out in https://github.com/dod-ccpo/atst/blob/master/config/base.ini#L14

Comment (leigh-mil) [reply atst/domain/auth.py]: @drummond-dds I just tested it out and it did not cause a regression on the previous bug!

**Pull request merged**